src/integration/pages/onDemandRadioEpisode/pashtoBrand/amp.test.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @service pashto
+ * @pathname /pashto/bbc_pashto_radio/p0340yr4
```

Comment:
Should this be the `/pashto/bbc_pashto_radio/programmes/p0340yr4`?

Reply:
No, it is correct as is

Reply:
Also, should the brand tests live within the episode folder - I am wondering if there is a slight reshuffle which would be a bit better e.g.

![image](https://user-images.githubusercontent.com/58214768/82462733-dadd9680-9ab3-11ea-8f51-6556d5786940.png)

Reply:
> No, it is correct as is
https://www.test.bbc.com/pashto/bbc_pashto_radio/p0340yr4 = 404.
https://www.test.bbc.com/pashto/bbc_pashto_radio/programmes/p0340yr4 = radio brand page

https://www.bbc.com/pashto/bbc_pashto_radio/p0340yr4 = PAL radio brand page

I just want to make sure that the routes are consistent across all environments

Reply:
The data is stored in bbc_pashto_radio/p0340yr4 on local

The actual routing is https://www.test.bbc.com/pashto/bbc_pashto_radio/programmes/p0340yr4

https://www.test.bbc.com/pashto/bbc_pashto_radio/p0340yr4 should 404 as it is not the correct path

Reply:
The URL in local should ideally match that of the live route (irrespective of where the local data is located - e.g. TC2 MAPs - the local url will always match the live route, but where we serve the data from is totally different).

I just don't want to have to start using different URLs in local vs test vs live - it's too confusing.

Still don't know why PAL returns a page for this https://www.bbc.com/pashto/bbc_pashto_radio/p0340yr4 if the URL is technically incorrect - one for Jane?

Reply:
The URL in local is http://localhost:7080/indonesia/bbc_indonesian_radio/programmes/w13xtt0s

Reply:
I think I have figured out what the deal is:
- https://www.bbc.com/pashto/bbc_pashto_radio/p0340yr4 is a radio episode
- https://www.bbc.com/pashto/bbc_pashto_radio/programmes/p0340yr4 is a radio brand

They happen to have the same ID (and because the page data is very similar, this just works). 🤯

Reply:
`p0340yr4` is a brand ID - it's not fully working on the PAL episode page - there is no player

PAL just reacts in this way when you pass a brand ID to the episode page - there is no 404, just a weird half-working page

You can verify this with other brand IDs too - eg `indonesia/bbc_indonesian_radio/w13xtt0s`